src/SDKs/ServiceBus/data-plane/tests/Microsoft.Azure.ServiceBus.Tests/Microsoft.Azure.ServiceBus.Tests.csproj

```diff
@@ -11,28 +17,11 @@
 
   <ItemGroup>
     <Compile Remove="obj\**" />
```

Comment:
Any idea who adds bin and obj to these item groups?

Reply:
They had existed in the original project file and Visual Studio (2017 and especially 2019) was rather insistent on including them in the project without specific removal directives.

Reply:
FWIW these projects should no longer have an obj folder under it as they should be redirected under a root artifact\obj folder instead.

Reply:
Yeah, that's what I would have expected;  unfortunately, they keep reappearing and getting sucked into the project.    I'll give them a scrub for this submission, but if they continue to be a hassle for local work, I'll add them back.

Reply:
OK yeah it you are fighting against VS just let it be, although it is kind of annoying.

Reply:
VS is refusing to not generate local `obj` and `bin` - and continues to suck `obj` in.   I removed the `bin` stuff, but had to keep `obj`

Reply:
Are there any files going into the local obj directory?

Reply:
Only on Visual Studio builds (both 2017 and 2019).    Builds using `dotnet build` or `dotnet test` obey the overrides and find their way to the artifacts directory.   The `dotnet` operations do not regenerate the folders, as expected.